### PR TITLE
Created the basic skeleton for mzdb wrapper

### PR DESCRIPTION
**Pull request by theboywholived**
msdk-io-mzdb is the new module. 
I have only created the file MzDBFileImportMethod.java and the implementation of the RawDataFile for MzDB.

### CODE REVIEW

**Comment by tomas-pluskal** [comment]
It seems the build is failing, take a look at the Travis log.

**Comment by tomas-pluskal** [comment]
The pom.xml of your module looks a bit messed up, multiple dependencyManagement tags etc.

**Comment by tomas-pluskal** [comment]
Ideally, you should use similar pom.xml to the msdk-io-mzml module (for example), and only add the dependencies of the mzdb-access library. There is no need to modify the rest of the pom.xml

**Comment by theboywholived** [comment]
Ughh, similar errors were happening in Eclipse when I first merged the two POM.xml files. I then used Eclipse's recommendations to add it to the build path. 

I can't figure out how to merge the two pom.xml files. There are certain dependencies in mzdb-access contained in the ` <dependencyManagement> 
<dependencies> 
<dependency1>
 </dependency1> 
<dependency2>
 </dependency2> 
</dependencies> 
</dependencyManagement> ` 
 and certain dependencies only like these
 `<dependencies> 
<dependency1>
 </dependency1> 
<dependency2>
 </dependency2> 
</dependencies> 
`
I then added my dependencies in the dependencyManagement tag.

There are also multiple plugins in the pluginManagment tag, should those be ignored?

**Comment by tomas-pluskal** [comment]
Yes, ignore the plugins and move all the dependencies to one place (the same dependencyManagement tag).

**Comment by theboywholived** [comment]
If I do that, the closing dependencyManagement tag shows an error. Eclipse doesn't show what the error, even on hovering.

**Comment by tomas-pluskal** [comment]
Well, try to debug by removing different parts of the pom.xml to see where the error comes from. You can also run maven from the command line (mvn compile) to see what error it reports.

File: msdk-io-mzdb/src/test/java/io/github/msdk/io/mzdb/MzDBFileExportMethodTest.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package io.github.msdk.io.mzdb;
```
Comment (tomas-pluskal) [inline msdk-io-mzdb/src/test/java/io/github/msdk/io/mzdb/MzDBFileExportMethodTest.java]: Please add the license header (same that all other MSDK source files have)

File: msdk-io-mzdb/src/test/java/io/github/msdk/io/mzdb/MzDBFileExportMethodTest.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+/**
+ * Hello world!
+ *
+ */
+public class MzDBFileExportMethodTest 
+{
+    public static void main( String[] args )
```
Comment (tomas-pluskal) [inline msdk-io-mzdb/src/test/java/io/github/msdk/io/mzdb/MzDBFileExportMethodTest.java]: No need for main() function in junit tests.

File: msdk-io-mzdb/src/main/java/io/github/msdk/io/mzdb/MzDBFileImportMethod.java [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+          // SimpleChromatogram eachChromatogram = new SimpleChromatogram();
+          // eachChromatogram.setDataPoints(rtValues, mzList, intensityList, mzList.length);
+        }
+
+      } catch (StreamCorruptedException e) {
+        throw new MSDKException("Stream corrupted, detected by mzDB");
+      }
+
+    } catch (ClassNotFoundException e) {
+      throw new MSDKException("Class can't be found by mzDB");
```
[the remark this answers is not in the document]
Comment (theboywholived) [reply msdk-io-mzdb/src/main/java/io/github/msdk/io/mzdb/MzDBFileImportMethod.java]: Should I do this for all other exceptions or only for `ClassNotFoundException` ?

[the remark this answers is not in the document]
Comment (tomas-pluskal) [reply msdk-io-mzdb/src/main/java/io/github/msdk/io/mzdb/MzDBFileImportMethod.java]: All of them

**Pull request merged**